VideoLocker/res/layout/player_controller.xml

```diff
@@ -1,224 +1,224 @@
+
+        <org.edx.mobile.third_party.iconify.IconButton
+            android:id="@+id/lms_btn"
+            android:layout_width="20dp"
+            android:layout_height="20dp"
+            android:layout_marginEnd="12dp"
+            android:layout_marginRight="12dp"
+            android:background="@null"
+            android:contentDescription="@string/app_name"
+            app:iconColor="@color/edx_grayscale_neutral_white_t"
+            app:iconName="fa_share_square_o"
             tools:targetApi="17" />
 
-        <ImageButton
-                android:id="@+id/lms_btn"
-                android:layout_width="20dp"
-                android:layout_height="20dp"
-                android:layout_alignParentRight="true"
-                android:layout_alignParentEnd="true"
-                android:layout_centerVertical="true"
-                android:layout_marginRight="12dp"
-                android:layout_marginEnd="12dp"
-                android:background="@drawable/ic_lms_selector"
-                android:contentDescription="@string/app_name"
-                tools:targetApi="17" />
-
-        <ImageButton
+        <org.edx.mobile.third_party.iconify.IconButton
             android:id="@+id/share_btn"
             android:layout_width="20dp"
             android:layout_height="20dp"
-            android:layout_centerVertical="true"
-            android:layout_toLeftOf="@id/lms_btn"
-            android:layout_toStartOf="@id/lms_btn"
-            android:layout_marginRight="10dp"
             android:layout_marginEnd="10dp"
-            android:background="@drawable/ic_facebook_discuss_selector"
+            android:layout_marginRight="10dp"
+            android:background="@null"
             android:contentDescription="@string/app_name"
-            tools:targetApi="17"
-            />
-                
-    </RelativeLayout>
+            app:iconColor="@color/edx_grayscale_neutral_white_t"
+            app:iconName="fa_facebook_square"
+            tools:targetApi="17" />
+
+    </LinearLayout>
 
     <LinearLayout
         android:id="@+id/panel_bottom"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_alignParentBottom="true"
-        android:layoutDirection="ltr"
+        android:layout_gravity="bottom"
         android:background="#CC000000"
-        android:orientation="vertical"
-        android:padding="7dp" >
+        android:gravity="center_vertical"
+        android:orientation="horizontal"
+        android:padding="7dp">
+
+        <org.edx.mobile.third_party.iconify.IconButton
+            android:id="@+id/rew"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginEnd="8dp"
+            android:layout_marginLeft="4dp"
+            android:layout_marginRight="8dp"
+            android:layout_marginStart="4dp"

```

Comment:
`layout_marginStart` should be positioned along with `layout_marginLeft` or `layout_marginEnd` I guess.